flink-runtime/src/test/java/org/apache/flink/runtime/taskmanager/TaskCancelAsyncProducerConsumerITCase.java

```diff
@@ -84,7 +84,7 @@ public void testCancelAsyncProducerAndConsumer() throws Exception {
 			config.setInteger(ConfigConstants.LOCAL_NUMBER_TASK_MANAGER, 1);
 			config.setInteger(ConfigConstants.TASK_MANAGER_NUM_TASK_SLOTS, 1);
 			config.setInteger(TaskManagerOptions.MEMORY_SEGMENT_SIZE, 4096);
-			config.setInteger(TaskManagerOptions.NETWORK_NUM_BUFFERS, 8);
+			config.setInteger(TaskManagerOptions.NETWORK_NUM_BUFFERS, 16);
```

Comment:
just a note for the curious:  this test can cope with higher number of network buffers and is waiting for all of them to be blocked - increasing this to `9` would have been enough here though (we require 2 exclusive buffers now per default, while 1 was the minimum per incoming channel)

Reply:
yes, i will set 9 for it.